src/examples/design/icons/SvgSearch.tsx

```diff
@@ -142,8 +149,11 @@ export const SvgSearch: React.FC<ISvgSearchProps> = ({ data, searchEnabled }) =>
                 margin-bottom: ${p => p.theme.space.lg};
               `}
             >
-              <Label>Search icons</Label>
```

Comment:
Any particular reason why this was removed?

Inputs are usually accompanied by labels so that screen readers can identify the form control to assistive devices. The input has an `aria-labelledby` that points to an `id` that doesn't exist in the DOM.

https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input
https://webaim.org/techniques/forms/controls#input

Reply:
Oh, interesting. I didn't know the `Label` accompanies the screen readers. I'll reach out to Bethany about why the `Label` isn't present `MediaInput`.

Reply:
Some inputs don't have visible labels (e.g. https://google.com). But the DOM should still include a label for screen readers. Also, the accessibility tree should make sense (e.g. `aria-labelledby` should not point to a non-existant id).

Reply:
Ohh gotcha. There's an option to have the `Label` hidden via props.
Thanks for catching that Henry and hinting me to the solution. 😆